src/components/sync-configurator/BaseSyncConfigurator.jsx

```diff
@@ -152,6 +153,7 @@ class BaseSyncConfigurator extends React.Component {
                         size="large"
                         onClick={this.onSynchronize}
                         data-test="list-action-bar"
+                        disabled={disabled}
```

Comment:
Wouldn't a hidden make more sense? For the FAB only

Reply:
Let's recap the options:

1) Keep the button enable and show a snackbar message
2) [what we have here] disable the button
3) Hide the button

I have no strong opinion about this, I'll leave the decision for @adrianq

Reply:
I think leaving the greyed out button there is better, at least at first glance you can know there is some action you can perform on that page that is currently disabled.

Reply:
Maybe with styles we could darken it differently.

But I still prefer adding a snackbar message without disabling completely (UX wise) or a hide/show animation when selection happens.

Reply:
I don't have a strong opinion between disabling or showing a snackbar. One of the problems now is that the icon is not easily visible when disabled and can be a bit confusing for the user. Anyway, I would go for a snackbar and keep the same styling always. We can show a message telling the user to select at least one OU. Can you please take care of this @matiasgarcia91 ?

In case anybody is interested in the (non-existent) consensus https://ux.stackexchange.com/questions/83645/hiding-vs-disabling-actions-and-show-a-message-on-mobile

I will merge this https://github.com/EyeSeeTea/d2-ui-components/pull/27 anyway as it could be useful in the future